config/fafnir/default.toml

```diff
@@ -1,5 +1,32 @@
+[fafnir]
+  # Connection string to the postgreSQL containing POI data.
+  postgres = "postgresql://gis:gis@localhost"
```

Comment:
`postgres_url` feels more expressive in my mind

Reply:
It would even be more consistent with `elasticsearch` to have `postgres.url` instead of `fafnir.postgres_url`, even though there won't be any other fields related to postgres